statistics/table.go

```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/pingcap/tidb/util/tracing"
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
+	"golang.org/x/exp/maps"
```

Comment:
You could use the non-experimental package after we upgrade to 1.21.

Reply:
The standard library `maps` doesn't provide `Keys()`.